charts/jaeger/templates/agent-ds.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.agent.enabled -}}
+{{- if and .Values.agent.enabled (not .Values.allInOne.enabled)  -}}
```

Comment:
> `.Values.allInOne.enabled`

I don't think you should have made this change. I simply meant to change the defaults in values.yaml, not to introduce these massive changes across the chart.